src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -1370,6 +1378,70 @@ void ViewProviderSketch::editDoubleClicked()
     }
 }
 
+void ViewProviderSketch::toggleWireSelelection(int clickedGeoId)
+{
+    Sketcher::SketchObject* obj = getSketchObject();
+
+    const Part::Geometry* geo1 = obj->getGeometry(clickedGeoId);
+    if (isPoint(*geo1) || isCircle(*geo1) || isEllipse(*geo1)) {
```

Comment:
should this check for closed loop bsplines too or is that overkill?

Reply:
That would make sense.